src/vfs/public.c

```diff
@@ -160,10 +199,28 @@ bool vfs_mkparents(const char *path) {
 }
 
 char* vfs_repr(const char *path, bool try_syspath) {
-	char buf[strlen(path)+1];
-	path = vfs_path_normalize(path, buf);
-	VFSNode *node = vfs_locate(vfs_root, path);
+    char buf[strlen(path)+1];
+/*
+    void * buffer[100];
+    int nptrs = backtrace(buffer, 100);
+    char ** strings;
+    strings = backtrace_symbols(buffer,nptrs);
+    if(strings != NULL){
+        //functions are 0~nptrs
+        log_debug("functions list");
+        for(int j=0; j<nptrs; j++){
+            log_debug("%s",strings[j]);
+        }    
+    } 
```

Comment:
I recommend compiling with `meson configure -Db_sanitize=address,undefined` for debugging; you won't have to resort to hacks like these in 99% of cases.